utils/build.config

```diff
@@ -9,7 +9,7 @@ isal=v2.30.0
 isal_crypto=v2.23.0
 spdk=v22.01.2
 ofi=v1.19.1
-mercury=v2.4.0rc5
```

Comment:
@soumagne If the RPM requires libfabric >= 1.20, why doesn't the source build?
https://github.com/daos-stack/mercury/pull/122

Reply:
because it's a separate PR :)

Reply:
Why though? If we want to use this mercury update, do we need to also use the libfabric update?

Reply:
On aurora we use the system libfabric so doesn't matter, but outside of Aurora?

Reply:
because the CI does not build against aurora rpms :) you asked me: "If the RPM requires libfabric >= 1.20, why doesn't the source build?" that is the reason. You surely don't need the libfabric update if you build from source on aurora and use the system libfabric...

Reply:
and yes once mercury is built against libfabric >= 1.20, there's no going back because libfabric has an ABI breakage that prevents you from using an older libfabric once you compiled against libfabric >= 1.20.

Reply:
I mean there is a disconnect (putting Aurora aside)
RPMS: requires new mercury and new libfabric.
Source build for the exact same version of daos: requires new mercury but allows older libfabric.

I.e. the source build says this is allowed
```
ofi=v1.19.1
mercury=v2.4.0
```

But the mercury=v2.4.0 RPM says ofi >=1.20 is required.

So, building mercury from source vs installing mercury from RPM has different dependencies on libfabric versions. So what happens if we build THIS PR from source? We will get an incompatible OFI?

Reply:
right there is a difference between binary compatibility and source compatibility. Ideally we want to update both so that mercury can pick up new flags exposed by the newer libfabric.